src/lat/determinize-lattice-pruned.h

```diff
@@ -155,11 +155,14 @@ struct DeterminizeLatticePhonePrunedOptions {
   bool word_determinize;
   // minimize: if true, push and minimize after determinization.
   bool minimize;
+  // extra parameters for speed 
+  int max_loop;
+  int max_arcs;
   DeterminizeLatticePhonePrunedOptions(): delta(kDelta),
                                           max_mem(50000000),
                                           phone_determinize(true),
                                           word_determinize(true),
-                                          minimize(false) {}
+                                          minimize(false) , max_loop(-1), max_arcs(-1) {}
```

Comment:
I use it somewhere else in my recognizer which directly set the max_arcs and max_loops on the DeterminizeLatticePhonePrunedOptions struct . this improve the speed on DeterminizeLatticePhonePrunedWrapper